e2e-tests/cypress/e2e/pages/insurance/policy-and-export/typeOfPolicy.js

```diff
@@ -0,0 +1,38 @@
+import { FIELD_IDS } from '../../../../../constants';
+
+const { INSURANCE } = FIELD_IDS;
+
+const typeOfPolicy = {
+  intro: () => cy.get('[data-cy="intro"]'),
+  [INSURANCE.POLICY_AND_EXPORTS.POLICY_TYPE]: {
+    single: {
+      label: () => cy.get(`[data-cy="${INSURANCE.POLICY_AND_EXPORTS.SINGLE_POLICY_TYPE}-label"]`),
+      input: () => cy.get(`[data-cy="${INSURANCE.POLICY_AND_EXPORTS.SINGLE_POLICY_TYPE}-input"]`),
+      hintList: {
+        item1: () => cy.get(`[data-cy="${INSURANCE.POLICY_AND_EXPORTS.SINGLE_POLICY_TYPE}-hint-list-item-1"]`),
```

Comment:
can we dereference this on top since there's a lot of repetition of same variable